src/styles/blocks/card.scss

```diff
@@ -0,0 +1,68 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid $quaternary-color;
+  border-radius: 5px;
+  transition: transform 0.3s ease-in;
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__img {
+    text-align: center;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__name {
+    margin-top: 40px;
+    margin-bottom: 4px;
```

Comment:
you shorthand

Reply:
Not fixed
<img width="1014" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/86789161/61433160-81e7-4a5a-8fd3-43fa5770d9b6">